.github/workflows/cont_integration.yml

```diff
@@ -89,13 +89,13 @@ jobs:
       matrix:
         blockchain:
           - name: electrum
-            features: test-electrum
+            features: test-electrum,verify
           - name: rpc
-            features: test-rpc
+            features: test-rpc,verify
```

Comment:
This is not necessary as rpc doesn't verify transactions

Reply:
Right, missed that..